Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -17,26 +17,22 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Länderübergreifende Risiko-Ermittlung um Schweiz erweitert</item>
-        <item>Änderung der Risikokarten</item>
+        <item>Check-in per QR-Code für effiziente Rückverfolgbarkeit von Infektionsketten</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
-        <item>Die Corona-Warn-App nutzende Personen können nun auch verschlüsselte Zufalls-IDs mit Personen austauschen, die die offizielle Warn-App der Schweiz nutzen. Somit können Warnungen an App nutzende Personen in der Schweiz geschickt sowie von ihnen empfangen werden.</item>
-        <item>Auf den Risikokarten entfällt die Anzeige der Anzahl aktiver Tage bzw. dass die App dauerhaft aktiv ist.</item>
+        <item>Die CWA ermöglicht nun einen Check-in für Events und Orte. Personen, die Events veranstalten oder ein Geschäft haben, können über die App einen QR-Code erstellen. Durch Scannen des QR-Codes können sich Gäste bei Ankunft einchecken, um so ihre Anwesenheit zu registrieren. Auf Wunsch legt die App außerdem einen entsprechenden Tagebuch-Eintrag an.Wird eine eingecheckte Person später positiv auf das Coronavirus getestet, werden alle Personen automatisch gewarnt, die zur selben Zeit eingescheckt waren.</item>
```

Comment:
There is a space missing here: "...an.Wird..." - apologies, this was already wrong in Figma

Reply:
done :)